wire-ios-request-strategy/Tests/Sources/Services/SupportedProtocolsServiceTests.swift

```diff
@@ -134,10 +140,34 @@ final class SupportedProtocolsServiceTests: MessagingTest {
         }
     }
 
+    private func createSelfUser(in context: NSManagedObjectContext) -> ZMUser {
+        let selfUser = ZMUser.selfUser(in: context)
+        selfUser.remoteIdentifier = UUID()
+
+        return selfUser
+    }
+
+    private func createClient(for user: ZMUser, in context: NSManagedObjectContext) -> UserClient {
+        let client = UserClient.insertNewObject(in: context)
+        client.user = user
+        client.remoteIdentifier = UUID().uuidString
+
+        return client
+    }
+
+    private func createSelfClient(in context: NSManagedObjectContext) -> UserClient {
+        let selfUser = createSelfUser(in: context)
+        let client = createClient(for: selfUser, in: context)
+
+        context.setPersistentStoreMetadata(client.remoteIdentifier, key: ZMPersistedClientIdKey)
+
+        return client
+    }
```

Comment:
non-blocking: this could be moved to `ModelHelper`

Reply:
Thanks, will make a follow up to this, because all tests passed and it's mergeable now.